### PR TITLE
Fix/TAO-7216/Timer doesn't start after the audio is fully loaded when the connection is slow 

### PR DESCRIPTION
**Pull request by bziondik**
DRAFT PR!!!

Related to: https://oat-sa.atlassian.net/browse/TAO-7216
Handling warning from ItemRunner in case of errors during Item rendering

Related to https://github.com/oat-sa/extension-tao-itemqti/pull/1279

### CODE REVIEW

File: views/js/controller/runner/runner.js [hunk 1]
```diff
@@ -171,6 +171,24 @@ define([
                 }
             };
 
+            /**
+             * Handles warnings
+             * @param {String} [displayMessage] - an alternate message to display
+             */
+            var onWarning = function onWarning(err, displayMessage) {
```
Comment (jsconan) [inline views/js/controller/runner/runner.js]: `onError` and `onWarning` are very similar. Could you refactor them in order to share the same mechanism between the rwo?

File: manifest.php [hunk 2]
```diff
@@ -39,7 +39,7 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '33.2.1',
+    'version'     => '33.2.2',
```
Comment (jsconan) [inline manifest.php]: Could you also mention the dependency with taoQtiItem? (`20.0.2`)

File: views/js/controller/runner/runner.js [hunk 3]
```diff
@@ -171,6 +171,24 @@ define([
...
+                displayMessage = displayMessage || err.message;
+
+                if(!_.isString(displayMessage)){
+                    displayMessage = JSON.stringify(displayMessage);
+                }
+
+                logger.error({ displayMessage : displayMessage }, err);
```
[the remark this answers is not in the document]
Comment (bziondik) [reply views/js/controller/runner/runner.js]: fixed

File: views/js/controller/runner/runner.js (hunk 1 above)
Comment (bziondik) [reply views/js/controller/runner/runner.js]: refactored

**Pull request merged**